puffin_egui/src/flamegraph.rs

```diff
@@ -502,7 +516,7 @@ fn paint_timeline(
     // We show all measurements relative to start_ns
 
     let max_lines = canvas.width() / 4.0;
-    let mut grid_spacing_ns = 1_000;
+    let mut grid_spacing_ns = (options.grid_spacing_ms * 1_000.) as i64;
```

Comment:
The math here is wrong. There are 1M ns for each md.

Reply:
I get where you're coming from, but with he correct math it doesn't work. `16_666` ns (the old value) creates an interval of 16.666ms.

Easiest fix might just be to change it to `_micros` since the unit itself doesn't really need to be exposed I think.